regression-test/data/nereids_p0/expression/fold_constant/fold_constant_by_be.out

```diff
@@ -2,21 +2,6 @@
 -- !sql_1 --
 80000
 
--- !sql --
-PLAN FRAGMENT 0
-  OUTPUT EXPRS:
-    sleep(100)[#0]
-  PARTITION: UNPARTITIONED
-
-  HAS_COLO_PLAN_NODE: false
-
-  VRESULT SINK
-     MYSQL_PROTOCAL
-
```

Comment:
this out result is empty, this is excepted?

Reply:
yes